shared/js/bluetooth_helper.js

```diff
@@ -175,11 +175,6 @@ var BluetoothHelper = function() {
     },
 
     getConnectedDevicesByProfile: function(profileID, cb) {
```

Comment:
I don't have strong opinion on naming, but if it's my call, I will say keep it without change. Thanks.